public/reference/accounting-faculty.html

```diff
@@ -160,7 +160,7 @@ <h5>Associate Dean of Graduate Studies and Research
                         Ph.D. Program Director, Professor of Accounting
                      </h5>
                   </div>
-                  <div class="accounting-faculty-img-cntnt">

```

Comment:
another specific class name that we probably don't need, don't see any styles attached

Reply:
on second inspection, we should probably keep public/reference/* untouched as possible